.ci-mgmt.yaml

```diff
@@ -29,3 +29,11 @@ actions:
       run: |
         cd provider && go test -v -json -count=1 -cover -timeout 2h -tags=${{ matrix.language }} -parallel 4 . 2>&1 | tee /tmp/gotest.log | gotestfmt
 pulumiConvert: 1
+
+worktreeAllowedChanges: |-
+  sdk/go/random/pulumi-plugin.json
```

Comment:
It might not be obvious for the next reader _why_ these two files can have changes. Could you comment?